src/pages/Setup/LightningDialog.tsx

```diff
@@ -23,7 +23,7 @@ const images = {
   [SetupLightning.NULL]: "",
 };
 
-const LightningDialog: FC<InputData> = ({ callback }) => {
+export default function LightningDialog({ callback }: Props) {
```

Comment:
I prefer named exports over defaults :P  
Makes it clearer/more obvious when importing.

Reply:
Would be inconsistent since we always used dedault exports for the pages